gse-util/src/main/java/com/powsybl/gse/util/NodeChooser.java

```diff
@@ -48,6 +54,9 @@
     private boolean success;
     private Set<String> openedProjects = new HashSet<>();
     private SimpleBooleanProperty deleteMenuItemDisableProperty = new SimpleBooleanProperty(false);
+    private Optional<CopyService> copyService;
```

Comment:
Don't use global copyService since it will depend on the filesystem of the selected nodes